app/app.go

```diff
@@ -244,7 +250,7 @@ type App struct { // nolint: golint
 	ScopedIBCParentKeeper capabilitykeeper.ScopedKeeper
 
 	// the module manager
-	mm *module.Manager
+	MM *module.Manager
```

Comment:
I needed this for the purposes of disabling distribution during the test suite. This can be returned to "unexposed once we split the consumer and provider apps"